internal/driver/driver.go

```diff
@@ -138,7 +139,15 @@ func generateReport(p *profile.Profile, cmd []string, vars variables, o *plugin.
 
 	// Output to specified file.
 	o.UI.PrintErr("Generating report in ", output)
-	out, err := os.Create(output)
+
+	openOutput := func(output string) (io.WriteCloser, error) {
+		if o.Writer != nil {
```

Comment:
Rather than have a condition here, should the code always use the writer since it's initialized to oswriter{} in https://github.com/google/pprof/blob/db0be723d40dfbb90c702d493a71d398173358e7/internal/driver/options.go#L38?

Reply:
Yep, sure! I missed It.